File: 7-kyu/easy-wallpaper/python/solution.py
from math import ceil

def wallpaper(l, w, h):
    number_words = ["zero", "one", "two", "three", "four", "five",
               "six", "seven", "eight", "nine", "ten",
               "eleven", "twelve", "thirteen", "fourteen", "fifteen",
               "sixteen", "seventeen", "eighteen", "nineteen", "twenty"]
    if not (l > 0.0 and w > 0.0 and h > 0.0):
        return number_words[0]
    index = int(ceil(((2.0 * w * h) + (2.0 * h * l)) * 1.15 / 5.2))
    return number_words[index]

File: 7-kyu/easy-wallpaper/javascript/solution.js
const wallpaper = (l, w, h) => {
  const number_words = ["zero", "one", "two", "three", "four", "five",
    "six", "seven", "eight", "nine", "ten",
    "eleven", "twelve", "thirteen", "fourteen", "fifteen",
    "sixteen", "seventeen", "eighteen", "nineteen", "twenty"];
  if (!(l > 0 && w > 0 && h > 0)) return number_words[0];
  const index = Math.ceil(((2.0 * w * h) + (2.0 * h * l)) * 1.15 / 5.2)
  return number_words[index]
}